v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -121,6 +127,8 @@ public interface KafkaToBigQueryFlexOptions
 
   @TemplateParameter.Text(
       order = 9,
+      parentName = "avroFormat",
```

Comment:
Can we separate this(Avro, confluent wire format, schema registry) etc to a different class? In that way, it can be used for other kafka templates? If its not trivial, we can do it later.

Reply:
I tried refactoring, but I am running into some issues so I will postpone this for now.